cob_phidgets/ros/src/phidgetik_ros.cpp

```diff
@@ -222,7 +224,7 @@ auto PhidgetIKROS::update() -> void
 		if(_indexNameMapItr != _indexNameMapDigitalIn.end())
 			name = (*_indexNameMapItr).second;
 		names.push_back(name);
-		states.push_back(this->getInputState(i));
+		states.push_back(!this->getInputState(i)); // Inverted by the board for unknown reasons
```

Comment:
So has this been wrong all the time? Or is this only happening on some boards?
@ipa-bnm where on raw's or cob's do we use this/can we verify this?